OpenRA.Mods.Common/Traits/BotModules/Squads/States/AirStates.cs

```diff
@@ -21,7 +21,7 @@ abstract class AirStateBase : StateBase
 	{
 		static readonly BitSet<TargetableType> AirTargetTypes = new BitSet<TargetableType>("Air");
 
-		protected const int MissileUnitMultiplier = 3;
+		protected const int MissileUnitMultiplier = 1;
```

Comment:
This multiplier will do nothing we set to `1`. Is there any point in keeping it?

Reply:
I don't know?  I leave it here because I believe devs won't like hardcore threat different in code,  even if it can help perform better only for now.  Maybe later we should make Squad manager receive Multiplier and maybe the individual air threat for a list of units or sth.

Reply:
Leaving dead code is an antipattern. If this is ever need again we can easily just reintroduce it.

Reply:
ok,  if you say so I will remove it for good